src/logics/BVLogic.cc

```diff
@@ -9,7 +9,6 @@ Module: New Logic for BitVector
 #include "SStore.h"
 #include "PtStore.h"
 #include "BVLogic.h"
-#include "CUFLogic.h"
```

Comment:
Do we need BV logic at all? Or is it used in some specific cases?

Reply:
We don't, support for bit-vectors should be rewritten. But we might still re-use some code from BVLogic and BitBlaster, so we can keep them for now.